docs/developers/tooling/libraries/viem.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Viem
+image: /img/socialCards/use-linea.jpg
+---
+
+Viem is TypeScript Interface for Ethereum, Build reliable apps & libraries with lightweight, composable, and type-safe modules that interface with Ethereum.
+
+As the authors of wagmi, a popular React Hooks library for Ethereum, we struggled to work with the existing low-level TypeScript Ethereum libraries. We wanted to provide the users of wagmi with the best possible developer experience, <br/>
+**Viem**: a TypeScript Interface for Ethereum that provides low-level stateless primitives for interacting with Ethereum. An alternative to ethers.js and web3.js with a focus on reliability, efficiency, and excellent developer experience.
```

Comment:
this doesn't really follow on that well from the above statement. Maybe add a connecting statement like "Due to our experiences, we developed Viem [...]".